library/general/src/lib/config_file/config_file.rb

```diff
@@ -0,0 +1,29 @@
+module ConfigFile
+  class ConfigFile
```

Comment:
In general, a matching FooModel and FooParser. Now, does `GrubConfModel` describe /etc/grub.conf? `EtcDefaultGrubModel`? But that's not important for the general API.